shared/message/Geometry.proto

```diff
@@ -58,7 +58,7 @@ message Frustum {
 }
 
 message Cone {
-    fvec3 axis     = 1;
-    float gradient = 2;
-    float radius   = 3;
+    fvec3 axis = 1;
+    // Cos(theta) of the cone angle
+    float radius = 3;
```

Comment:
Should we update the index?

Reply:
Would mean old NBS files won't play properly... Do we have any we care about?

Reply:
good point, we probably do have some